ci/run-k8s-e2e-tests.sh

```diff
@@ -174,12 +174,14 @@ function run_sonobuoy() {
     if [[ "$focus_regex" == "" && "$skip_regex" == "" ]]; then
         $SONOBUOY run --wait \
                 $KUBECONFIG_OPTION \
+                --e2e-parallel=true \
```

Comment:
Running whole-conformance shouldn't use parallel mode, if https://github.com/antrea-io/antrea/pull/849/ is correct.

Reply:
Updated, thanks.